collectors/diskspace.plugin/README.md

```diff
@@ -1,6 +1,6 @@
 # diskspace.plugin
 
-This plugin monitors the disk space usage of mounted disks, under Linux.
+This plugin monitors the disk space usage of mounted disks, under Linux. To be able to get the statistics, Netdata needs to have the search permission on the mount point.
```

Comment:
I'd suggest rewording this new sentence to:

`This plugin requires Netdata to have execute permissions on the mount point itself, as well as each component of the absolute path to the mount point.'

This explicitly spells things out in a way that a user who doesn't have a bunch of UNIX background should understand and be able to translate into an actual fix, whereas 'search permission' is something most people are going to have to look up (even I had to think for a minute to realize that it was talking about the execute bit being set in the directory permissions).  Also, I think it's important to point out that the permission is needed for each component of the path to the mount point, not just the mount point itself.

Reply:
Why not both, as in "have search permissions (set the `execute` bit, see [man chmod](https://linux.die.net/man/1/chmod))"

Reply:
That works too, though I do think it's still worth mentioning that those permissions are needed on each path component.  It's something that someone who's experienced with UNIX-like systems will already know, but I regularly see it catch people by surprise who do not have lots of experience.

Reply:
Sure, I wasn't suggesting to remove the "each component" part.